assets/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "engine-strict": true,
   "engines": {
-    "node": "^10.16",
+    "node": "^12.14",
```

Comment:
Shall we go straight to 12.15+ if we are to upgrade it?

Reply:
`12.14` was the lastest LTS when I started this PR… I’ll see if it’s possible to upgrade to `12.15`

Reply:
Because of this: https://snyk.io/blog/node-js-release-fixes-a-critical-http-security-vulnerability/

Reply:
Thanks, ✅